### ISSUE TITLE
UX run command directly from minion page

### ISSUE THREAD

**Issue by dawidmalina** [erwindon/SaltGUI#105]
I think it would be nice improvement if we could run command directly from the minion page by adding new option to existing menu:
<img width="922" alt="zrzut ekranu 2018-12-27 o 16 23 48" src="https://user-images.githubusercontent.com/5910851/50485116-39ecc800-09f4-11e9-8ce6-c02591497e38.png">

which will simply route to run command page with filled minion name:
<img width="1263" alt="zrzut ekranu 2018-12-27 o 16 25 03" src="https://user-images.githubusercontent.com/5910851/50485135-512bb580-09f4-11e9-8112-8783f565e5c1.png">

**Comment by erwindon** [comment]
So basically, it is the same as "Sync state..." in which you type another command?
I was doubting about the same feature earlier. But I found it to be superfluous because it is so easy to update the `Sync state...` dialog box with an alternative command.
When an explicit `Run...` command was added, it would have to be present on other pages as well. e.g. because you would want to run a command from the "grains" category that just happens to be unsupported with the current gui.
On the other side, the above is just a workaround.

**Comment by erwindon** [comment]
I decided to make separate pages to make the UI less crowded (than alternative solutions) when extending SaltGUI with all that extra functionality. SALT is a 2-dimensional world: (1) the list of minions and (2) the list of operations. I found that grouping the functionality was most practical. Since there is only little room to present a summary for a minion, the pages look very similar. A practical solution is to add direct access to the grains/pillars/schedules of minion from the main screen. That would also save some page-loading.

The command was always called "Sync state...", at least from before I joined the project. Strange that I also never noticed the confusing wording. My only excuse is that I never needed the actual 'sync' commands. I'll rename the command anyway to "Apply state..." (done).

I really appreciate your input, let there be no doubt about that!
